src/arith_uint256.cpp

```diff
@@ -10,6 +10,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include "crypto/common.h"
```

Comment:
Maybe this is the wrong place for this discussion. But better here then nowhere.
IMO clean code leads to good software quality in general.
Once the code gets a mess, the quality in general will follow.
"A broken window is the start of a vagabonded house". Thats why i honor @Diapolo work on one hand.

But:
1.)
Nowadays, code-structure can be handled by tools like clang-format. There is no need for nits for spaces, indents, etc.
Even if the clang-format output is not stable, i think a re-formatting from time-to-time over the whole code makes sense and will prevent us from thousands of nits and gives us more time to focus on the "real stuff".

2.)
Alphabetic ordering of includes just don't make sense. Someone does it, someone not. I think we don't need nit's for this.

3.)
Cleanup commits like #5616 are okay, but will break (requires rebase) all other and upcoming pull requests in this area. That's why i would recommend to keep these kind of pull request to a minimum. Better provide a large cleanup commit once (at best in a code-quiet time).
To much of cleanup-commit will slow down important work.